tests/VeresOne.spec.js

```diff
@@ -70,6 +70,65 @@ describe('methods/veres-one', () => {
         .to.eql(JSON.stringify(UNREGISTERED_DOC, null, 2));
     });
 
+    it('should return a key present in an un-registered DID', async () => {
+      nock('https://ashburn.capybara.veres.one')
+        .get(`/ledger-agents`)
+        .reply(200, LEDGER_AGENTS_DOC);
+
+      const {ledgerAgent: [{service: {ledgerQueryService}}]} =
+        LEDGER_AGENTS_DOC;
+      nock(ledgerQueryService)
+        .post('/?id=' + encodeURIComponent(UNREGISTERED_NYM))
+        .reply(404);
+
+      _nockLedgerAgentStatus();
+
+      // eslint-disable-next-line max-len
+      const unregisteredKey = 'did:v1:test:nym:z6MkesAjEQrikUeuh6K496DDVm6d1DUzMMGQtFHuRFM1fkgt#z6MkesAjEQrikUeuh6K496DDVm6d1DUzMMGQtFHuRFM1fkgt';
+      const result = await v1.get({did: unregisteredKey});
```

Comment:
@dmitrizagidulin The one thing I don't like about this API is that the value you're passing for `did` here is *not* a DID. That's why I think `url` or something should be the parameter name ... and if `did` is used, it should probably throw an error if it's not a DID. But that can be addressed in the future.

Reply:
I agree. I think I went so far as put it an alias, so you could either say `.get({did})` or `.get({url})` for this very reason, but now I can't find it - I think it was at another library. But, agreed.